src/Analyzer.Cli/CommandLineParser.cs

```diff
@@ -143,7 +143,11 @@ private void SetupCommonOptionsForCommands(List<Command> commands)
 
                 new Option(
                     "--include-non-security-rules",
-                    "Run all the rules against the templates, including non-security rules")
+                    "Run all the rules against the templates, including non-security rules"),
+
+                new Option<FileInfo>(
+                    "--custom-rules-json-path",
+                    "The rules JSON file to use against the templates. If not specified, will use the default rule set that is shipped with the tool.")
```

Comment:
Similarly here: `"The JSON rules file ..."`

Reply:
Done